crates/aptos/src/common/init.rs

```diff
@@ -159,34 +158,21 @@ impl CliCommand<()> for InitTool {
         };
         let public_key = private_key.public_key();
 
+        let client = aptos_rest_client::Client::new(
+            Url::parse(profile_config.rest_url.as_ref().unwrap())
```

Comment:
I don't like this `unwrap` on `rest_url`. If we're assuming that this will always be Some then let's just not make it an Option at all.

Reply:
This is an option, because it's a YAML file that it reads from directly (in serialization).

If it is *not* an option, then it will load poorly from the config and give a very cryptic deserialization error.

Earlier in the code, it enforces that the REST URL is set (it's either preset, or creates one).

It's the unfortunate circumstance of this structure being used in multiple places and not being able to assume that the REST URL is set always because someone could manually modify the file, and therefore mess it all up.